discoverygarden/xml_forms
# 8.x.elems

D8 port of remaining elements

# What does this Pull Request do?

This pull request ports the remaining custom form elements.

# How should this be tested?

Create a form with one of each of the following elements. 

CreativeCommons
DatepickerXmlFormElements
DefaultableItem
DefaultableMarkup
Indent
Inline

Verify the same behavior present in D8 and D7.
Verify both object creation and metadata edits.
